### ISSUE TITLE
pre-training models

### ISSUE THREAD

**Issue by jilner** [nnUyi/ReMAEN#2]
Thank you for thr job,when will pre-training models of other data be available?

**Comment by nnUyi** [comment]
Yeah, I am busy preparing for AAAI recently. All the pre-trained models will released later.

**Comment by jilner** [comment]
Thank you for thr job,when will pre-training models of other data be available?
